health_vara.py

```diff
@@ -257,6 +310,47 @@ def get_most_recent_imaging_request_datetime(self, _name):
             dt = max(i.date for i in self.imaging_test_requests)
             return dt
 
+    @classmethod
+    def search_most_recent_imaging_request_datetime(cls, _name, clause):
```

Comment:
Adds specific searching by the most recent datetime:

![image](https://github.com/vara-ai/health_vara/assets/1367955/d9c3cec8-e406-40b5-84a2-f381a104c030)